src/components/visualizations/styles/tooltip.scss

```diff
@@ -86,72 +86,90 @@
     }
 }
 
-.gd-viz-tooltip-tail {
-    position: absolute;
-    left: 50%;
-    width: 0;
-    height: 0;
-    margin-left: -10px;
-    border-width: 10px;
-    border-style: solid;
-    pointer-events: none;
-
-    &.right {
-        left: calc(100% - 14px);
-    }
+.gd-viz-tooltip-item {
+    display: block;
+    margin-bottom: 10px;
 
-    &.left {
-        left: 14px;
+    &:last-child {
+        margin-bottom: 0;
     }
 }
 
-.gd-viz-tooltip-tail1 {
-    bottom: -20px;
-    border-color: transparentize($bubble-light-border-base, 0.5) transparent transparent transparent;
-}
-
-.gd-viz-tooltip-tail2 {
-    bottom: -18px;
-    border-color: #fff transparent transparent;
+.gd-viz-tooltip-title {
+    display: block;
+    overflow: hidden;
+    font-size: 12px;
+    font-weight: normal;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    text-align: left;
+    color: $gd-color-label;
 }
 
-.gd-viz-tooltip-table {
-    max-width: 320px;
+.gd-viz-tooltip-value-wraper {
+    padding-right: 1em;
 }
 
-.gd-viz-tooltip-table-row {
-    &:last-child .gd-viz-tooltip-table-cell {
-        border-bottom: none;
+.gd-viz-tooltip-value {
+    position: relative;
+    display: block;
+    overflow: hidden;
+    font-size: 14px;
+    line-height: 1.3em;
+    max-height: 2.6em;
+    text-align: justify;
+    margin-right: -1em;
+    padding-right: 1em;
+
+    &:before {
+        content: "...";
+        position: absolute;
+        right: 0;
+        bottom: 0;
     }
-}
 
-.gd-viz-tooltip-table-cell {
-    border-bottom: 10px solid transparent;
-
-    &:empty {
-        display: none;
+    &:after {
+        content: "";
+        position: absolute;
+        right: 0;
+        width: 1em;
+        height: 1em;
+        margin-top: 0.2em;
+        background: white;
     }
 }
 
-.gd-viz-tooltip-table-title {
+.gd-viz-tooltip-value-max-content {
+    display: block;
     overflow: hidden;
-    max-width: 140px;
-    font-size: 12px;
+    font-size: 14px;
+    height: 0;
     font-weight: normal;
-    line-height: 14px;
     white-space: nowrap;
-    text-overflow: ellipsis;
-    text-align: right;
-    vertical-align: middle;
-    color: $gd-color-label;
+    text-align: left;
+    visibility: hidden;
 }
 
-.gd-viz-tooltip-table-value {
-    overflow: hidden;
-    max-width: 165px;
-    padding: 0 0 0 5px;
-    font-size: 14px;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    vertical-align: top;
+.multiline-supported {
+    .gd-viz-tooltip-value-wraper {
+        padding-right: 0;
+    }
+
+    .gd-viz-tooltip-value {
+        display: -webkit-box;
```

Comment:
is it possible to keep 'display', '-webkit-line-clamp' and '-webkit-box-orient' props next to each other or the best in separate class to explicitly say that they need to be set together to allow clamping to work?